causing/graph.py

```diff
@@ -187,10 +187,13 @@ def remove_node_keep_edges(graph, node):
     """Keep transitive connections when removing node.
 
     Removing B from A->B->C will result in A->C.
+
+    WARNING: This function only approximates the edge effects. To get accurate
+    results, you must shrink the model and recalculate the effects, instead.
```

Comment:
Thanks for pointing that out. We will need that even in the final version, right?